src/amo/components/Search/index.js

```diff
@@ -26,12 +28,12 @@ export class SearchBase extends React.Component {
     count: PropTypes.number,
     dispatch: PropTypes.func.isRequired,
     enableSearchSort: PropTypes.bool,
+    errorHandler: PropTypes.object.isRequired,
     filters: PropTypes.object,
-    hasSearchParams: PropTypes.bool.isRequired,
+    filtersFromState: PropTypes.object,
```

Comment:
Yeah actually, I think that's a better name anyway. I think you suggested it first and I thought this would be okay but I agree, it's not great. Maybe `filtersUsedForResults` if that's not too verbose.